Sources/PubNubChatComponents/Theming/MessageListComponentTheme.swift

```diff
@@ -52,16 +48,14 @@ public class MessageListComponentTheme: ViewControllerComponentTheme {
     incomingItemTheme: MessageListCellComponentTheme,
     authorItemTheme: MessageListCellComponentTheme?,
     typingIndicatorCellTheme: TypingIndicatorCellTheme,
-    enableReactions: Bool
+    reactionTheme: ReactionTheme? = nil
```

Comment:
by default it should be enabled

Reply:
It's a breaking change for the current requirement. I'm not saying we should revert it, but if we all agree on that then it should be explicitly mentioned in documentation that there's no longer flag like `enableReactions` and it's replaced with `reactionTheme`

Reply:
thats true, but when it is still prerelease version we do not care about it so much (as we spoke with @CraigLn yesterday on daily)